### DISCUSSION TITLE
RFC 0006: opening discussion (standard overlay changes)

### DISCUSSION THREAD

**Discussion by blelump** [the-human-colossus-foundation/oca-spec#62] [General]
The [RFC 0006](https://github.com/the-human-colossus-foundation/oca-spec/blob/master/RFCs/0006-Standard_overlay_changes.md) proposes significant changes to the `standard overlay` . Here's the discussion thread to cover various aspects of the proposed changes.

**Comment by carlyh-micb** [comment]
The core change is this idea:

"Standards MUST be referenced by the required value standard_id which should use (where available) persistent identifiers to reference the standard. Examples of persistent identifiers include URNs, PURLs, DOIs, and SAIDs. Other details about the standard are optional (standard_label, standard_location, and standard_version) and can help users identify and use the correct standard."

Is there a reason this should not be implemented? Technically?
